README.md

```diff
@@ -62,6 +62,28 @@ const startedContainer = await container
   .start();
 ```
 
+Building your own Docker name with custom name/tag and skip if already built:
+
+```javascript
+const path = require("path");
+const { GenericContainer } = require("testcontainers");
+
+const buildContext = path.resolve(__dirname, "dir-containing-dockerfile");
+
+const container = await GenericContainer.fromDockerfile(buildContext)
+  .withImageName("my-image")
+  .withImageTag("my-tag")
+  .skipBuildOnExistingImage()
```

Comment:
I think we're missing something here with the design because `skipBuildOnExistingImage` only has an effect if you provide one of `withImageName` or `withImageTag`. 

I wonder if we could better reflect that. For example:

```typescript
const container = await GenericContainer.fromDockerfile(buildContext)
  .withImageBuilder()
    .withName("my-image")
    .withTag("my-tag")
    .skipBuildOnExistingImage()
    .build()
  .withBuildArg("arg-key", "arg-value")
  .build()
```

I also wonder if it's really a use-case that anyone would want the build to fail if there already exists that container name and tag, maybe it's not required?

Reply:
Very good idea!

Reply:
Done! I also forwarded the `Uuid` to the nested builder because it is only used there. I still made it passable by the parent constructor, to not break existing logic ;)